src/Helpers/AliasEmitters/AliasCollection.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Collect aliases.
+ *
+ * This class collects aliases, which may be provided in any order.
+ * It then sorts them and returns the result when requested.
+ */
+
+namespace Pantheon\Terminus\Helpers\AliasEmitters;
+
+/**
+ * Collect aliases
+ */
+class AliasCollection
```

Comment:
This logic belongs in `Pantheon\Terminus\Collections`, if it is kept. I think this logic should be held by the Environments collection (Pantheon\Terminus\Collections\Environments), which you would request an Alias model (here called AliasData) from, which would be a model belonging to each Environment.

Reply:
I could probably re-use some of the collection code in Terminus core; I didn't do that because I thought that the collections were all optimized to be built by querying the API, which is not done here. I'll look at it and see what can be re-used.

I cannot make the Alias Model belong to an Environment, though. That would be way too slow.

Reply:
All of the current code that references environments can be removed. This used to be used in some instances; however, now the code always uses a wildcard alias record, which means there is always exactly one alias "environment" per site, `'*'`.

Reply:
After removing environments, I might be able to make the Alias Model belong to the site, since I already am creating Site objects for every site, I believe.